docs/blog/2020-02-17-changing-the-oss-status-quo/index.md

```diff
@@ -0,0 +1,35 @@
+
+The status quo is made and remade every time we as maintainers add to our documentation. Every time we, as maintainers, engage with the community via social media, issue queues, and our documentation, we help decide who contributes. We contribute to their safety and their productivity. Right now, barriers to contribution are severely limiting our capacity to create and sustain open source projects.
+
+There's an opportunity here. There are thousands of highly skilled potential contributors ready, willing, and able to contribute to open source projects but they need to know that they'll be safe in doing so. A handful of projects are intentionally focusing on this work and making incredible strides towards a more inclusive open source community.
+
+Gatsby's most admirable quality is the company's commitment to that community. Yes, it's a great tool but what use is any tool that is too difficult to use or too obscure for most people to access? One of Gatsby's core values is "You Belong Here". We back it up with a clear code of conduct that we consistently reference and enforce. The documentation is vast, includes specifics around contributing, and provides materials that make it easy for people outside of the core team to teach as well as learn. New contributors who have had a PR merged in are invited to the GitHub organization and offered SWAG as a thank you. There are even a limited number of one-on-one mentorship opportunities available.
+
+Consider your own community and adopt the strategies that are feasible for your project. Who comes back to contribute more than once? Who is heard when making decisions as a group? Who feels empowered to speak publicly about your project? When interpersonal issues arise, who is believed? Take the time to examine your project's processes and documentation. What's missing? What's out-of-date? Tear down as many barriers to entry as you can identify. The more people are encouraged and prepared to participate, the more likely it becomes that you'll be able to share the burden of maintenance across a wider team.
+
+At the same time, listen to people in marginalized groups. Show up and shut up. Support individual contributors and affinity groups. Participate in those groups where your presence doesn't introduce a safety concern. (Ask if you have any doubts, and maybe even if you don't.) You could do this by sponsoring an event like the [Blacks in Technology Conference](https://bitcon.tech/) or attending a local Lesbians Who Tech event. Once you're there, avoid centering your own experience. Absorb as much as you can from the experiences of others and educate yourself on how to be the best ally possible. Keep in mind that you don't get to _decide_ that you are an ally. We all have to earn trust over time.
```

Comment:
```suggestion
At the same time, listen to people in marginalized groups. Show up and take a seat. Support individual contributors and affinity groups. Participate in those groups where your presence doesn't introduce a safety concern. (Ask if you have any doubts, and maybe even if you don't.) You could do this by sponsoring an event like the [Blacks in Technology Conference](https://bitcon.tech/) or attending a local Lesbians Who Tech event. Once you're there, avoid centering your own experience. Absorb as much as you can from the experiences of others and educate yourself on how to be the best ally possible. Keep in mind that you don't get to _decide_ that you are an ally. We all have to earn trust over time.
```
It's less alliterative, I know–but does "take a seat" have the same impact as "shut up"? That might align better with Gatsby's code of conduct.

Reply:
I take your point! I chose it less for the alliteration and more because that specific phrase is commonly used to communicate that an aspiring ally should start by listening. I'll fiddle with it but will likely change the phrase entirely rather than swap out part of it.